pkg/pxc/app/binlogcollector/binlog-collector.go

```diff
@@ -173,7 +175,7 @@ func GetDeployment(cr *api.PerconaXtraDBCluster, initImage string) (appsv1.Deplo
 		Spec: appsv1.DeploymentSpec{
 			Replicas: &replicas,
 			Selector: &metav1.LabelSelector{
-				MatchLabels: labels,
```

Comment:
while doing this, we can also consider labelling the Deployment of binlog collector.

Reply:
@hors